app/models.py

```diff
@@ -46,6 +37,17 @@ def post_redflag(self):
             "message":"Your Report Has Been Saved Successfully"
         }  
 
+    def get(self, id):
+        self.id = id
+        response = self.model.find_by_id(self.id)
+        return make_response (jsonify({"My red-flag" : response}), 200)
+        record = [record for record in redflag_records if record['Id'] == id]
+        if len(record) == 0:
+             = jsonify({
```

Comment:
Why do you have this equal sign before jsonify

Reply:
I don't have the equal sign in my local file. Trying to resolve the conflicts and update it to show the changes.

Reply:
Great